base_layer/wallet/src/output_manager_service/storage/sqlite_db/mod.rs

```diff
@@ -881,10 +888,18 @@ impl OutputManagerBackend for OutputManagerSqliteDatabase {
                     &conn,
                 )?;
             } else if output.spent_in_tx_id == Some(i64::from(tx_id)) {
+                debug!(
```

Comment:
maybe warn or info level? or is that too much - do these happen often?

Reply:
You are right, on the transaction service side when a transaction is rejected the message is a warn. However these will fire when you do a manual cancellation so I think I will make these Info rather.